Content.Shared/Alert/AlertAutoRemoveComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Alert;
+
+/// <summary>
+///     Copy of the entity's alerts that are flagged for autoRemove, so that not all of the alerts need to be checked constantly
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
+public sealed partial class AlertAutoRemoveComponent : Component
+{
+    [ViewVariables]
+    [AutoNetworkedField]
+    public Dictionary<AlertKey, AlertState> Alerts = new();
```

Comment:
Should be a datafield so it supports persistence, also you never use the after auto handle state so remove the true from the autogencompstate attribute.